config.json

```diff
@@ -1,5 +1,5 @@
 {
-	"targetOrganisation": "ahmdigital",
+	"targetOrganisation": "octokit",
```

Comment:
are we keeping this as ahmdigital or octokit?

Reply:
Can keep AHM, but i was using octokit for testing, ecause there's more rows in the table.